src/loaders/prowizard/prowiz.c

```diff
@@ -132,10 +130,7 @@ int pw_wizardry(HIO_HANDLE *file_in, FILE *file_out, const char **name)
 		goto err2;
 	}
 
-	if (hio_error(file_in)) {
-		/* reset error flag */
-	}
-
+	hio_error(file_in); /* reset error flag */
```

Comment:
Same with this one. https://github.com/libxmp/libxmp/commit/4781ec44162a6ed548a96386f92cb0b27f85498e